docs/user-guide/user-manual/travel-time-matrices.md

```diff
@@ -137,12 +123,10 @@ destinations["geometry"] = destinations.geometry.centroid
 
 With this, we have all input data sets needed for computing a travel time
 matrix: a transport network, origins, and destinations. We still need to decide
-which modes of transport should be used, and when to depart.
+which modes of transport should be used, and the departure time in our analysis.
 
 The modes of transport can be passed as a list of different
-{class}`r5py.TransportMode`s (or their {class}`str` equivalent), the departure
-must be a {class}`datetime.datetime`. If you search for public transport routes,
-double-check that the departure date/time is covered by the input GTFS data set.
+{class}`r5py.TransportMode`s (or their {class}`str` equivalent). Meanwhile, the departure must be a {class}`datetime.datetime`. If you search for public transport routes, double-check that the departure date/time is covered by the input GTFS data set in the `calendar.txt` file.
```

Comment:
This is maybe too specific and detailed. We have an own paragraph on which tools to use for checking a GTFS data set on the ‘required data’ page, I’ll add a link to there, instead. Not sure all users are aware of the structure of a GTFS archive